extensions/blocks/testimonial/editor.scss

```diff
@@ -0,0 +1,121 @@
+.wp-block-jetpack-testimonial {
+    display: flex;
+    flex-direction: column;
+}
+
+.wp-block-jetpack-testimonial__content {
+    .is-aligned-left & {
+        text-align: left;
+    }
+    .is-aligned-center & {
+        text-align: center;
+    }
+    .is-aligned-right & {
+        text-align: right;
+    }
+}
+
+.wp-block-jetpack-testimonial__author {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    
+    .is-aligned-right & {
+        justify-content: flex-start;
+    }
+    
+    .is-aligned-center & {
+        justify-content: center;
+    }
+    
+    .is-aligned-right & {
+        justify-content: flex-start;
+        flex-direction: row-reverse;
+        text-align: right;
+    }
+}
+
+.wp-block-jetpack-testimonial__meta {
+    display: flex;
+    flex-direction: column;
+    color: #999;
```

Comment:
Should you use a color var here?

Reply:
It's definitely a placeholder. I think I'll need to run this by design folks whether to use something like opacity so it works universally across themes and dark/light variants or whether there is a variable to use. There definitely are vars in editor but we also need to figure out the color for the frontend and that might be different between themes.